src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/LicenceController.kt

```diff
@@ -314,4 +316,36 @@ class LicenceController(private val licenceService: LicenceService) {
   ) {
     licenceService.updateBespokeConditions(licenceId, request)
   }
+
+  @GetMapping(value = ["/staffId/{staffId}"])
+  @PreAuthorize("hasAnyRole('SYSTEM_USER', 'CVL_ADMIN')")
+  @Operation(
+    summary = "Licences by Staff Id",
+    description = "Find licences associated with a supervising probation officer. Can be filtered by licence status. Requires ROLE_SYSTEM_USER or ROLE_CVL_ADMIN.",
+    security = [SecurityRequirement(name = "ROLE_SYSTEM_USER"), SecurityRequirement(name = "ROLE_CVL_ADMIN")],
+  )
+  @ApiResponses(
+    value = [
+      ApiResponse(
+        responseCode = "200",
+        description = "Licence details returned"
+      ),
```

Comment:
Does this need the type including in order to generate the example response in swagger correctly? Noticed previously that it wasn't adding this when the 200 response was not present, or when the type wasn't specified. This is odd, since previous versions of swagger didn't need the explicit 200 response here. Worth a check locally?

Reply:
Yes, thank you, coming up